### PR TITLE
Added total amount feature issue#35

### PR DESCRIPTION
**Pull request by cijokb**

### CODE REVIEW

File: src/components/ViewServiceDetails/TotalAmount/TotalAmount.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    return totalAmount.toLocaleString('en-IN', {
+        maximumFractionDigits: 2,
+        style: 'currency',
+        currency: 'INR',
+    });
+};
+
+const TotalAmount = ({ data }) => {
+    return (
+        <Fragment>
```
Comment (ivinjose) [inline src/components/ViewServiceDetails/TotalAmount/TotalAmount.js]: Fragment is not really needed here. 

Also, following is more readable, right?

```
if( !data || data.length === 0) return null;
return(
<div className={styles['amount-cmp']}>
    Total Amount : {getTotalAmount(data)}
</div>
)
```

File: src/components/ViewServiceDetails/TotalAmount/TotalAmount.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            {data && data.length > 0 ? (
+                <div className={styles['amount-cmp']}>
+                    <div className={styles['amount']}>
+                        Total Amount : {getTotalAmount(data)}
+                    </div>
+                </div>
```
Comment (ivinjose) [inline src/components/ViewServiceDetails/TotalAmount/TotalAmount.js]: Why do we need 2 divs (amount-cmp and amount)?

File: src/components/ViewServiceDetails/TotalAmount/TotalAmount.js (hunk 1 above)
Comment (cijokb) [reply src/components/ViewServiceDetails/TotalAmount/TotalAmount.js]: done

File: src/components/ViewServiceDetails/TotalAmount/TotalAmount.js (hunk 2 above)
Comment (cijokb) [reply src/components/ViewServiceDetails/TotalAmount/TotalAmount.js]: replaced inner div with span.

**Comment by ivinjose** [comment]
closed #35

**Pull request merged**